docs/wiki/migrate-alz-policies-to-builtin.md

```diff
@@ -0,0 +1,153 @@
+
+- Go to https://portal.azure.com
+- Open Policy
+- Go to Definitions and in Search find the ALZ custom policy. 
+
+  ![alz-custom-policy-def-search](media/alz-update-to-builtin-01.png)
+
+- Click on the hyperlink for the policy definition
+- To determine if the policy is assigned at any scope in the ALZ management group structure start by getting the policy definition ID
+
+  ![alz-custom-policy-def-name](media/alz-determine-policy-assign-01.png)
+
+- Since there is no easy way to get the various scopes a policy is assigned to, go to Azure Resource Graph Explorer
+- Ensure that scope for the query is Directory and then execute the following kusto query:
+
+  ```kusto
+    PolicyResources | 
+    where kind =~ 'policyassignments' and tostring(properties.policyDefinitionId) =~ '/providers/Microsoft.Management/managementGroups/contoso/providers/Microsoft.Authorization/policyDefinitions/Deny-PublicIP'
+    | extend 
+        assignmentScope = tostring(properties.scope),
+        assignmmentNotScopes = tostring(properties.notScopes),
+        assignmmentParameters = tostring(properties.parameters)
+    | project assignmentScope,
+        assignmmentNotScopes,
+        assignmmentParameters
+  ```
+
+- The above command will give a result similar to what is shown below
+
+  ![alz-custom-policy-assignments](media/alz-determine-policy-assign-02.png)
+
+- As can be seen this particular policy is assigned with only a simple Deny effect parameter at the following levels in the management group structure
+  - Contoso/contoso-landingzones/contoso-landingzones-corp
+  - Contoso/contoso-platform/contoso-platform-identity
+
+> Note
+that the provided example has a simple parameter set. If more complex parameters are assigned to a policy which is to be migrated those should be noted down. In that respect the possibility to download the query results as CSV could be leveraged.
+
+- Switch from Azure Resource Graph Explorer back to the Policy view 
+- Change the scope to include the scopes determined in the previous step. and search for the relevant policy
+
+  ![alz-delete-policy-assignments](media/alz-delete-policy-assign-01.png)
+
+- For each assignment, click the ellipsis and select Delete Assignment.
+- Once all policy assignments are deleted, go to the Definitions pane, search for the definition. Once found click the ellipsis and choose Delete Policy Definition
+
+  ![alz-custom-policy-def-search](media/alz-update-to-builtin-01.png)
+
+- To assign the _Not allowed Resource types_ policy search for that policy definition. Once found click the ellipsis and choose Assign
+
+  ![alz-builtin-policy-def-search](media/alz-assign-builtin-policy-01.png)
+
+- Set relevant parameters, for this particular policy this would be the resource type to disallow, i.e. Microsoft.Network/publicIpAddresses, then assign the policy to the scopes previously determined.
+
+### Migrate ALZ custom policies in initiatives to built-in policies
```

Comment:
Is this the correct heading? As at the end of this we are still assigning a custom policy, right?